sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -199,7 +198,7 @@ def __init__(self, coder, step_label):
     self._step_label = step_label
 
   def _check_safe(self, value):
-    if isinstance(value, (str, six.text_type, long, int, float)):
+    if isinstance(value, (str, bytes, int, float)):
```

Comment:
<!--new_thread; commit:05e95b6995f0b88a53930c30459c8d81e6692219; resolved:0-->
I think we need to keep `long` for Python 2 compatibility.